lang/test/org/partiql/lang/eval/EvaluatingCompilerTests.kt

```diff
@@ -1475,4 +1535,68 @@ class EvaluatingCompilerTests : EvaluatorTestBase() {
                      listOf("a", "b"),
                      result.first().orderedNames)
     }
+
+    @Test
+    fun selectDistinct() = assertEval(
```

Comment:
Add tests which include `SELECT DISTINCT <expr> FROM ...` where `<expr>` is:

- string concatenation using `||` 
- function call with arguments
- arithmetic (such as `+`)